src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1510,6 +1515,38 @@ class spell_dru_wild_growth : public SpellScriptLoader
     }
 };
 
+// 210706 - Gore
+class spell_dru_gore : public AuraScript
+{
+    PrepareAuraScript(spell_dru_gore);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_GORE_PROC });
+    }
+
+    void HandleProc(AuraEffect* aurEff, ProcEventInfo& procInfo)
+    {
+        if (SpellInfo const* spellInfo = procInfo.GetSpellInfo())
+        {
+            if (spellInfo->Id == SPELL_DRUID_BEAR_THRASH || spellInfo->Id == SPELL_DRUID_SWIPE ||
```

Comment:
List of spells should be done using spell_proc table
SpellFamilyMask0 0x00200800 -- Maul (0x800) & Moonfire (0x200000)
SpellFamilyMask1 0x00100000 -- Swipe (Bear)
SpellFamilyMask2 0x08000000 -- Thrash (Bear)

Reply:
this spell has proc flags set on it and a server side area dummy, it looked like the perfect candidate for a script rather than spell_proc.

how do you know when a spell should be done using the spell_proc table vs checking it on script?

Reply:
The answer is always. If it can  be done with db, it should be done with db

Reply:
> 
> 
> The answer is always. If it can be done with db, it should be done with db

@Shauren, how can you know which part of flag from every spell should be taken to compose the result spell proc flag?
Swipe (Bear)
 flag [0] 0x00000000 [1] 0x00100000 [2] 0x00002000 [3] 0x00000000
Thrash (Bear)
flag [0] 0x00000000 [1] 0x00000000 [2] 0x08402000 [3] 0x00000000

For example, why did you propose SpellFamilyMask1 0x00100000 for Swipe (Bear) and SpellFamilyMask2 0x08000000 for Thrash (Bear) rather than SpellFamilyMask2 0x00002000 for both Swipe (Bear) and Thrash (Bear), or why not SpellFamilyMask2 0x08002000 for Thrash (Bear), and so on. 
Is there some rule for composing proc flags?